components/camel-sjms2/src/main/docs/sjms2-component.adoc

```diff
@@ -0,0 +1,128 @@
+## Simple JMS Component
+
+*Available as of Camel version 2.19*
+
+The Simple JMS 2.0 Component is an extension of the SJMS Component that adds support for JMS 2.0 features. See the link:../../../../camel-sjms/src/main/docs/readme.html[SJMS Component] for the full list of features and documentation.
+
+Maven users will need to add the following dependency to their `pom.xml`
+for this component:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>camel-sjms2</artifactId>
+    <version>x.x.x</version>
+    <!-- use the same version as your Camel core version -->
+</dependency>
+----
+
+### URI format
+
+[source]
+----
+sjms2:[queue:|topic:]destinationName[?options]
+----
+
+Where `destinationName` is a JMS queue or topic name. By default, the
+`destinationName` is interpreted as a queue name. For example, to
+connect to the queue, `FOO.BAR` use:
+
+[source]
+----
+sjms2:FOO.BAR
+----
+
+You can include the optional `queue:` prefix, if you prefer:
+
+[source]
+----
+sjms2:queue:FOO.BAR
+----
+
+To connect to a topic, you _must_ include the `topic:` prefix. For
+example, to connect to the topic, `Stocks.Prices`, use:
+
+[source]
+----
+sjms2:topic:Stocks.Prices
+----
+
+You append query options to the URI using the following format,
+`?option=value&option=value&...`
+
+### Component Options and Configurations
+
+
+
+
+// component options: START
+[NOTE]
```

Comment:
A pushed up what I have but I am having trouble with the docs it doesn't look like its generating the content and they are failing validation, I am probably missing something obvious.